tools/myst/enc/enc.c

```diff
@@ -180,8 +180,8 @@ static uint64_t _forward_exception_as_signal_to_kernel(
         // convention as myst_handle_host_signal is expected to be called
         oe_context->rsp = (rsp & -16) - 8;
         oe_context->rbp = rbp;
-        oe_context->rdi = (__typeof(oe_context->rdi))&siginfo;
-        oe_context->rsi = (__typeof(oe_context->rsi))&mcontext;
+        oe_context->rdi = (__typeof(oe_context->rdi)) & siginfo;
```

Comment:
I think the extra space will be remove by "make format".

Reply:
It looks like the whitespaces are being added by `code-format`. Maybe it thinks its the binary & operator?

Reply:
Haha!